src/pages/FriendShipPage/index.ts

```diff
@@ -1 +1,6 @@
+export { FriendShipPage } from "./FriendShipPage";
```

Comment:
Usually named exports are being used when you need to import multiple values from a single file. In this case we always have a single component. It's almost like you are destructuring an object with only 1 key. Just pointing out.

Reply:
_nods_ this was an explicit choice. I've read a few things over the years that suggest using default exports leads to more potential issues/etc. A few links from a quick skim of google:

- https://basarat.gitbook.io/typescript/main-1/defaultisbad
- https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad
- https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/

Obviously this speaks to a stylistic/etc choice, and we should look at the evidence for/against as a team (when we have time) and decide what we want our canonical choice here to be.